vendorconsent/bitfield20.go

```diff
@@ -0,0 +1,50 @@
+package vendorconsent
+
+import "fmt"
+
+func parseBitField20(data consentMetadata20) (*consentBitField20, error) {
```

Comment:
Naming is a bit confusing here, IMHO. I initially thought this was parsing bit field 20 instead of for TCF 2.0. Consider putting these into something like a `vendorconsent/tcf2` package.

Reply:
That would create 3 packages, a tcf1 package, a tcf2 package, and a general package to roll them together. It is doable, but a chuck or work.